### ISSUE TITLE
unable to run non docker method

### ISSUE THREAD

**Issue by billguo2022** [ItzCrazyKns/Perplexica#498]
Hi I cloned the repo then built and ran npm run start in both ui and src, i'm able to see it running, but then when i ask a question, i see this error: error: Error in discover route: connect ECONNREFUSED ::1:4000? does anyone know why? i also have searxng running using docker

**Comment by ItzCrazyKns** [comment]
Did you already have a pre existing instance of SearXNG? It shall start with the compose file

**Comment by billguo2022** [comment]
yes i have searxng and i ran docker-compose up and had it running in the background. And i ran npm run dev in both src and ui, but i kept getting errors, sometimes it's 409 sometimes it's 403

**Comment by ItzCrazyKns** [comment]
> yes i have searxng and i ran docker-compose up and had it running in the background. And i ran npm run dev in both src and ui, but i kept getting errors, sometimes it's 409 sometimes it's 403

You need to use the SearXNG file we provide and need to change the config for the SearXNG URLs as well.

**Comment by billguo2022** [comment]
and run npm run dev for ui and src then searxng in the background right?
